runtime/src/snapshot_package.rs

```diff
@@ -224,15 +224,15 @@ pub struct SnapshotPackage {
     pub slot: Slot,
     pub block_height: Slot,
     pub hash: SnapshotHash,
-    pub snapshot_storages: Vec<Arc<AccountStorageEntry>>,
-    pub status_cache_slot_deltas: Vec<BankSlotDelta>,
-    pub bank_fields_to_serialize: BankFieldsToSerialize,
-    pub bank_hash_stats: BankHashStats,
-    pub accounts_delta_hash: AccountsDeltaHash,
-    pub accounts_hash: AccountsHash,
-    pub epoch_accounts_hash: Option<EpochAccountsHash>,
-    pub write_version: StoredMetaWriteVersion,
-    pub bank_incremental_snapshot_persistence: Option<BankIncrementalSnapshotPersistence>,
+    pub(crate) snapshot_storages: Vec<Arc<AccountStorageEntry>>,
+    pub(crate) status_cache_slot_deltas: Vec<BankSlotDelta>,
+    pub(crate) bank_fields_to_serialize: BankFieldsToSerialize,
+    pub(crate) bank_hash_stats: BankHashStats,
+    pub(crate) accounts_delta_hash: AccountsDeltaHash,
+    pub(crate) accounts_hash: AccountsHash,
+    pub(crate) epoch_accounts_hash: Option<EpochAccountsHash>,
+    pub(crate) write_version: StoredMetaWriteVersion,
+    pub(crate) bank_incremental_snapshot_persistence: Option<BankIncrementalSnapshotPersistence>,
```

Comment:
Yeah your arguments about visibility make sense to me. As for whether the crate reorganization would help, I think it would. But doesn't seem super high priority right now.